### ISSUE TITLE
Some flying mounts are summoned in ground-only conditions

### ISSUE THREAD

**Issue by RetroFrito** [xod-wow/LiteMount#147]
What happens? Timelost Proto Drake and Bloodgorged Hunter are summoned as ground mounts

What did you expect to happen instead? These should be flying mounts

Did it work in a previous version of LiteMount? If so, what was the last version that worked? Not sure

Please do not modify anything below this line.

--- General ---

date: Sun Apr 16 10:35:57 2023
build: 10.0.7 | 48999 | Apr  6 2023 | 100007
version: 10.0.12
locale: enUS
current profile: Default

--- Player ---

name: Fritobandito-AeriePeak
class: HUNTER
level: 70
race: Worgen
faction: Alliance
spec: 1 253 Beast Mastery

--- Location ---

map: Zereth Mortis (1970)
mapPath: Zereth Mortis (1970) -> The Shadowlands (1550) -> Cosmic (946)
instance: Zereth Mortis (2374)
zoneText: Zereth Mortis
subZoneText: Path of Inception
IsFlyableArea(): false

--- Debugging Output ---

Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
PreClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
Dispatching rule LeaveVehicle
Dispatching rule Dismount [nofalling]
 - setting action to dismount
PreClick ok time 0.12489998340607
PostClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
 - setting action to in-combat action
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event PLAYER_TALENT_UPDATE
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event PLAYER_TALENT_UPDATE
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event BAG_UPDATE_DELAYED
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event PLAYER_TALENT_UPDATE
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event PLAYER_TALENT_UPDATE
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event BAG_UPDATE_DELAYED
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
PreClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
Refreshing status of all mounts.
Dispatching rule LeaveVehicle
Dispatching rule Dismount [nofalling]
Dispatching rule CopyTargetsMount
Dispatching rule ApplyRules
 - checking 18 rules for button 1
Dispatching rule Mount [class:HUNTER] Hunter
 - filters: Hunter
 - setting action to mount Champion's Treadblade
 - found matching rule 16
PreClick ok time 28.765200018883
PostClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
 - setting action to in-combat action
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event PLAYER_TALENT_UPDATE
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event PLAYER_TALENT_UPDATE
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event BAG_UPDATE_DELAYED
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
PreClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
Refreshing status of all mounts.
Dispatching rule LeaveVehicle
Dispatching rule Dismount [nofalling]
Dispatching rule CopyTargetsMount
Dispatching rule ApplyRules
 - checking 18 rules for button 1
Dispatching rule Mount [class:HUNTER] Hunter
 - filters: Hunter
 - setting action to mount Time-Lost Proto-Drake
 - found matching rule 16
PreClick ok time 29.011900007725
PostClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
 - setting action to in-combat action
PreClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
Dispatching rule LeaveVehicle
Dispatching rule Dismount [nofalling]
 - setting action to dismount
PreClick ok time 0.12749999761581
PostClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
 - setting action to in-combat action
PreClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
Dispatching rule LeaveVehicle
Dispatching rule Dismount [nofalling]
Dispatching rule CopyTargetsMount
Dispatching rule ApplyRules
 - checking 18 rules for button 1
Dispatching rule Mount [class:HUNTER] Hunter
 - filters: Hunter
 - setting action to mount Turbostrider
 - found matching rule 16
PreClick ok time 21.979499995708
PostClick handler called on LM_B1 (inputButton=LeftButton, isDown=true)
 - setting action to in-combat action
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Updating swim times due to MOUNT_JOURNAL_USABILITY_CHANGED.
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED
Got refresh event MOUNT_JOURNAL_SEARCH_UPDATED

--- Options DB ---

vYvwZXXns6Fl7tE2y94TqHRc(nvI6YsIsbBkRyMjChQezrYEv3DPTpSSMh0V99lbYmr1YY2Cv4WSyvaj
Y7laWLMLxUCXv3nSJ(5fJ7MwVE)UPpF14xE3F)lV7bJ7wHNE94WhYJBA7nRU9Nh3TF102LNNwEy5IF64
(dR2((HvRh)89Dopz1T3oE4Ep8N)PvB)WNVg)V77m(PJx)5RUB6J33XV4UvRxpUE427UVZ4Xekru(TJ7
UVZ5PJRxpD90T3EVP8hVB1HP3pS9A8J77C6hpChyy3EFh)z7oU66BaQnui(lVBA3QJB(Y763nD4UXD3n
nD9F9C))9u)hdBwD7WHH7nAo(PpF94vR2mC4ERB(YXdJ33fad)21GxVM(((JB2mT9Hth3Eyp(9ZJ(2M4
YZn4)ATPuB(rBNX3MWJ(LN7DrVn)22yYfcLX2zngAA4loJNESfpgTbBdEKaBkM)whdyce0eDTUiENdJk
5mPiHIN7Inmk4CnD5xzAJT0KWanX2mYGP0yAsD0pXyAcTM8Ariqx(PuBRT8UUUOldq8B4zAcEcgjtWcO
2cWAtwtbSMMO3eisLqLcli2r0tGORwJJXoGsGJGpeAmzkJiQMgpVswxxkQd1uyv2umJYUmJORJzGwNLq
iNnJz(qQaIoBYLjat0eABfGziulVIrtNJarxM7BBi2oXhITDeNZ6iKRjVAencwJTnZfXIqCzBMIOzzYR
eGDbLJ(qMozjArKtueZ6TEtz6(uWtpr8MGd8Ji9ljhXfikIMn(ritLngBmplWJ7szkdRUJe3nfmfaGOl
xBkdF8RUKnMktIMFrBPl2rZk6lOhJPHgxMujbfjYltZM6AkQbbtr1jt1(yWvuOSDHS2bH4Xan5SUzlut
lSCGynKQdWDxxgh9EAaMostQTJHrzWyHzLHqOjlxsfnexgBi01x(SnfSfcJy7SMGbQHeENHuk1u0vA9y
DzRrqo0t(mdPHSrTDmNMGFtQOmxGxWxWRuxHIkwGeatTnmaB8Eck0Yskdfwb4vKgrQqNeBNKCUyQyBc)
aeRKeNTnnMyHKmwVJwysTdg)XIUl0(7etRwBwF1JpfIz)h8iAzBEBxwg2wyf53wS)Tm6c2MpuSFD2ejt
AlgiDTfObxn(IKpawjTAS7fxMvxm38z)r0eITXiRQLG8SS2etMD30yB6mcAAIfZdRRn21KhajqllmK3m
3jaQG1jHgNLSYDKvsc(EiaekgGgw5H8KuMyRh2(EgfSM8th(gUXxSA7(ddBVAm712GuhgwVAZ1t7Ofm0
rreqUo3oT9lV7HRo8zIz6XBx8YFgHiwVEfn1V8UNmSB3Q90GEX4VoU(lVlBzdZIfxEh(8lh(eX8cniw7
UPpmIXTab(WxwC142HDRMisbM3iFQdh3HptbVF(WUH)9DdBZRz6VEnZ8ie0zX5JRpChb1TF3E81HvL8n
E60dozbBi87Lt3E8lV7z7xJug(Y7odjLHunEWUXHmWaL(OH9hgju65idHRN2SpdlWb3GeGwrl)fJxpD8
9h(MRbznICLgUE6tBMOb3FC3QbGSpzhcvE9EsP0lSPNGLfGzbfbwOtsOsOXt2TAZoGDahh)40bYXf4MV
gynwPbI7MeW8ZRMwpEGWL1xtubCjT4nRV(4g8lgh4pp7QXR2n9PIiD4AI3936(bZ)jnyy)T4NhwdWmDZ
xE3JisNq3l)0Q1RU9Ud0qGpnKp66TJdfEXBN2DljsFkYy4WWTJFfdaaSFnsRzZWraOfiLrOqSA74(mPr
Y1p8zq4xDh(9osfHjcKc3vddz9q4NzXtNG4(MvxrFAFMVti4R2DlymBOHz6GVNYKVy6WHC(CVCfPK93k
IxGTjqJEcJEZHBh2DnLFZQTBhUAnXNBiDhKz)06PnmRVdgElEZ2V72jG6xCe2kZHgXXmoxxjp1BYRfZr
l(rGQ4W)()bPSruHncb0)CChs2ei20UdR2NPTQrY469h2nTPOinoS7qHgFW)EKsomtHwEWNbog0xF4Dd
R2ohLCfHyhzj9RJBVUSAv5rRJu6t)GbAIdhoiY5haEX2pnoqkmGOXqE6aM((pb52CW3MbpPdr2nF3EyD
EI0od9lh2oqV)NoU92mJTlabZBh(8v7g3FGi0TK7fKhcg8CS4r)201zwfYDK8umCfCvC3ksmbD51tWjs
k7iGSN(7VMwJxoTnBF(zAwws(VcofUjBHTgcI95CiOCHh)4UJycxUBydz05YS9Dt7)EkjCSc9z97XRjh
gzdteRMsxF4dF3LWzcPW9Caekqyu0rHqC83orhYHWJlEmeKtKYoBb6rSayQsscImpqZ91qT9izcqrvx8
pj741zANYadCFuuPOHV62vBhoK9xBmWJpHuJzwimmaKFiXwbjV(eDZSYPxDk8YxD5tF0fV4vN9O)J)JS
iXr8XDWoCgsT4W0UnKfcSWphe3D7GkuEz8vCCfXeHT)rIPNYgSJB(465W5Ngiz2IXDFCCl5YGYqCHP5h
AyE9JN29(vxF9y25jS8bI)qup51fqGK1j1yKjm4mp44v3bxbWzz2HmeLpAZ4UbASKnWMVPx3acqV45Ke
aQddGlNvnQXtAX)y(sX386Ii3q5zdv71hiu7HdFEohnaCkdysT5e22LRYkAMocHrKHRh2c183cc6e3fD
KXtdYJasSJxT69cllZ1l(caBAXQXBh)kNBzIe1w9X5uzczcVOm3dthpqLqo(PSLac9pNX9JKOai6h(a4
uKIvgnaH(QpcojPyHriH5jhxK4)sYJ3z4nKLkQqJzxp6xh39E4nNILq5IT41WF(Xrs7JKZK2zECNp9H7
ocz0RU5g4gz1VosJNmCGhRnVNAaarGStU)(jEOmK3IlrmL7GLnzDo9jz8zU9zdB2os(P8ipUfNrQKKNG
3S9AGAWp8NZsciNE7DR2dkmhi4IXXBoX8OjBE081zKqjV6oX72RXcKJfeY6mJRNUD3raHlwDZHtWB4GF
5IUFO9h8NgwQH0zGZSpT(7ULeLi5IT5SlsgYVegmv4paf0IWYBPCoOCrUC1MC0)G)RDPI8v2Iafebyeb
ZJPO4pDydmoivdGQpjx2F12EKJjUGmLmufjiUmCXUB6QpKZQ4WDL45NnSdV4zzxEZOplPv9qitie4C4n
d5DmFaXS7xsTPhsKCMhC4kGiSa81iCfLTCIx7sxxOm0OsWaH(IjkcbLP5C7RP1F3VsdbQMq48ykFdoU1
40wYvTZFcX8qe8Ah9HwN3wvMifdsioZeRFc(sY(HJuAypE344DzN2nFJm)Y68QgoLd1bkFK3o8HS69CF
meoCy4QPnVF)3YdLNOdnhKsCfO4NYjQrPRoxVcjV)hz2CX4hbVJSgIKl6lzN4BhVkRlJc5n)15nNliPD
MLaSsglmVW8e4rc9u9aW37UHF73iBG8e2jPpN5pOqS5I2Zg3dk5edLGluIhdgn5gPHsr(XGkytZVsd8S
pnS7gczIT5ufVJJP2svlC89qp5fRUHcOonL9SHstHt4zPi(p2dVCNeEiLt0B70NGNS97PeU(zKlFoZ1Q
hclLQOfCLxD8a9kQCqh5Vz3hkzY)yIZxAd2h3nDZQ1JpF8Z7)lBkmi9BqMW)v9bUoS)Ow)whXFs3ENVA
)bn4ToK)KE6wh0FABCRd7pPZTZG1FwZARd7pS)S1HCVAj73A43Nr)h241za8pRxR1H9TBVQ(9QQuwpsE
nECd1yvKZiWZdR4A1d2s)4iZxxWZngHEBPhkuD)CBd5grMhAth3)hPrP5A1BDZ6zj3AIORr6BLj3gUCB
gsnDCt8InCdOYJ0Plk3btBWe4o8gcn(YBTMqu6)inax9TLXIut7AL2E56uGwqpBW2Q9cT0uxAGjBYleI
VrXjE6HUuGr(oFhp7yPtIuZXiVKLj3u62cpYKqXMgEaERPHbLRLBtx(LmEYTRuEBbaDUO8YuJX2wF0kG
QJB7f3hk57mZXHWwfa4yUyBke76ek2KuAVmmejj55EY5ATwg4TrP3Gal5fN4dTkFOmqVXzkVZfklI0iD
Pl2uNEzbIpjDr305BKvfV1wMsYABd1xAuujQeFNP(926Blp6tXeRVv2fG8NTbHu5gfAYcQGTYkyLmPvL
8SAvQMbkezmxjekTaxBYknNGt0Cbf3WgjsRMZbZ8cpfrYyEAQji9VdWKTMcrPnPqZu6shuhyTeUFWM8U
DWCuOp2e5vp6Z9oTqCsRvDjHnIpZOK0hY8i9m0JkVJiD2RrtKBcFgJyRwxNYpAmQtdN3xjyfnaVP8ORv
mXGLClRmNFmuhqbt9mb5JL1MAVldLuhysLpcHE5PGOOeuxCoN4xJ4eTcLAoTXNecKa57vQ2RRxOosMat
2gE7QQBbdrGMQ(mtkKSF2lJcP2jDvM7vp)9cOsrFOrx(Qile050PeEsXdvzM9nY9XNENZlomPTpZ7uGR
7yGms6ZX6KyDmxsDt3Ay2jVximrYqhjlv3fIKPmN6oAKNEbp5MEZ4bJ7EBG9ZoFB(SbxC2BJ1X2wFRq9
XwHHch(Sqq3(aezQkyzSpPBCIjHGaHko7vOhDkHYwdPgXycd1loYs(GesWbCMjAOOA1fTqiqvXlosJYU
aHh8mdhEZBRVKPy8vNe4Yi8jGtS(RVgmLIrirqBL4jeXhv8OcEMlMhzbp9PKAG2gz1fODMyRqlT)LrLL
igWElhHNOjxDKsUcM2o1(lwJLBf1iATuZCwJI2gUU6if(ysCiJ5md)LmiOndJjAxuIxtVSYZR0hRra1n
otI8izDZUuLpkCpJOTX7ribFsjrMnjNuMtzDKDGzPUrWLr2W8Pz7Vn9DJWX8sAdaM(k2j(XmqZMPou(C
Sst8J8UrkVmwhQ4sSXXzRyHLbJ2uFjLCyajO53L3RwZ8DMfQ(wEVKr4s2RuEKH63R0TyrrpQXbftc4dP
vx1KDMOucvrBPfB92kzRbFL1ecSSsLS5xmbWoKCA2s0NzQQAsNNpBsJq)CWQG3joNOz1vNvO(yuzl8Kk
BClbj4RUrbFST(OMnN47GmK5h7CQ(TVkw9vXMUF55x2PlFfN8ZMeRSgfpw0lzMgXEepNgNgQcoPLiHj1
LhV5453ICmzDHKjjPTdv8M6S81H6vemwFjRHMy(tE6U6NNnsjTDzoWStmaW)K4ZTQBCAojDjzrMhqVmj
iMLGHT5(WjtsYx031WXFPTivc9qJLxaeynvxl2)cTR4SsxqoVc5rMQJuL)Ivhbv3PBE7skIJpmBwscTP
uvQ6AQKGe)WJCNT1PXQ1b4LPklytqdcvL8vGnJjmBjKsAeraXJNHxs8pySmB(D68fNebJnnddyhsoNxK
3ee6QV1xFuIgBnDsctg5ajqFNvOHRlFfX0tvrtsCeNCTsEjOsPq9LZMLa2onIigqBJUSvwlhgsoheMto
at035vffpZjcd8JvtiijEPKab55W(gP6Szi11iX7bMefgBJx4fw6470wHqO(OyO30QLcNcbjt2oz1OdE
YCCOQuRf4RvtsUYQlMu3mfHsY3QX3u5rCsheokELdbo9g9SeL11dQSpyQgZ8QgHGLFhm)L8hG7LMOojw
EZhUhtwgl5uuo3t0tngrDto2ylZ7kHWGAIIttcrdveTtESTRIDSeagxY39EnroW1fvKOp2W5z2k1RMbv
uxvXvp4uY5YcwrYrxb64mrz0sNP5ZAZMK0GdKSGurjyeUA8hg8WrRejcQngL(cMkqzd(yRu6oavR4PHM
vBfc8BH8xm9sgRK2KrkONpppYK86KypS0Polw)ELbBRVmwPQzGscfhcZil2TF50FzYMGDM6KcArVojRb
QmHzSTccszsmBwscMifcgTIizhneH0xkIwyXcmDJv4RgMbjSnSfzB0ak(puhAxfOQTsRSQizkjcGXN0E
tbWMQdiwFKzrYPlBz9GqvpUFeYA608kA1qWb2ceUVD2zVTWm4J9LGGEfsD1rYF3h1uGSsnSq9iP1PrTP
tWjTxeiziBD9J1zj59jc788zEf02zXMR2JtcvMr(QnKju5AX26Y6RtRYlJs9gDnITdnGofU(Q6quvTJZ
g6SfJPgFAwCrPJlKwGeXVb(Xc60yUySv8dL)oJIXGKCwRXkLCzJnrnLmtqt)W2PnLfttI7fIQRq4lPkY
ewknwEA0qNb3Q6zNe7UPHnfmGJ1jsQUOWZTTjoReBB7SVphDLiIwPlquZyKKk8al5qioTj2WXTQuQU7R
NHV8KMXXl8UuumRJwRqHq0zRdStwOynPPM5yIOFkPic3BM4SfLDl4fqbKJDabSqQ1gH4zD280DkMi60r
jDccq8iH0wKv03la13fKMTdgT2R8uJtuJ9nIRAV0iK83D1zX0uxtukrapAveqI66WA1PKppsk)uMsTA6
LeuLqG8jWopGofdAGFzn)rTzhPKrSLNDMzHTdpw6uhZotBKn5a25sjkGzkf2chDQReJAPtaLLLrV42Y6
mcpMpvZMCGRoUXcijHKO1PD9H01yZqORkzZazT42gX9KAmrnkM2kuT1bOklA3EKAeqasLbHrYUx8EDNr
ah2PIajAulT7fn13YdakjwLbvZ(SRIQshnO3g13Qb(q5EIiSCeWPXsfblQlirzrAIm8nkXWp5qL9mhaz
oyeuiQL5G41sFcAQ91NAVm)s1TmnYU63LjLQUBK2AcRKoreHcE1MdOTrcGsuXOHkfAcZmLh7K2ZgM)oj
ecsIygOKO41R1GTv6PomN1EIGfneQdiub2SLqSJsIVCs0OfyQLYt3gGcJbUlBKp7Ku247VabjivyrOQd
qkUI)wexKlBH25drWlhq6L6nzGEKpCZ5Xc2MeIIAFA(r5O9VSunZSqR(6q5hHbzTWMaN9bzCBQdvsv0Q
TkcUj0IK6K(Qd)tw9cL4nc5qpYyJCntwwsMvT0BMP8izWG8(Ko(jx5c(T(6Jf0bHaJsKXOtAseSzKM7J
OZsXx4rjVS8wujsNwNyRy1wStHg4hTAMzzWw1VKq2bAxx5vaUrLIjLU7zASgnGMTARzuN7rr9lHCFziH
IkzPpXAvDwzl4G7unD0wr7JAoOS9ruDoITcy9ZsPVTI1fk03L6epZeFP(DjKPRvVdajP)vKoHW3agyQ5
uZPb7Cs)Pj7hjqfeOSdFQNJCAMTQba1qeg(oavHT1PB2mQe0PP0k7agY0pfQQYYMpImM1APqOwjL8yNe
YZj4Ib1TZPka0stldvOi6l6vajVPfm7K8xwEl13mn1sRA4RB3FBkk7WaLXQu3xJvcvRBJKCLxOytjPjf
O8WKgh1PLTLKMKGh1RqePUvWA(2uKFzC(EpjFwRvMVygM8zyxubHRsP7tbFNu)gPbiwVwzdai1vP6IGK
KobuMttvxOrw09YWRPIb(S0JDQ8c9jb9HgUuPgKjsHqrXdoL(gtEiCyst5vCFr2T1eXy9lAV(yLsahjW
Onj7abz3iU3anRb)dnHzwtskjAl3qsxTsGL6Emc1sN0rmNUceYYofT1TWed0ORL2iVAJtMF7EALnL1ol
7a42wYFIOHcrsBJj)smaD)e0l9fzVlHluXkfBTvWvTXtyo83P8)epMTswHzpMs4gKbCzbioVTIQXkuld
TCD4OpJQRLe8jmTRoYQ0wu2sZ2n7APi1BYJSh4Kbqn1hPTp2wXDCRMgirhAsmTskBQQiQkt6qEU8kMKG
oHe2SUrTKEvBn3l9SJqEoRlglkBLdrdzlvjFHtJ0lLD2N8NxwaYyL17PZ2sScugbOsIz3kOQpgQYTLIH
AuxFEKAj)ADK0WytfueOKGbjgnkk7RRy57EPuAtxDhncY(OA1stY8obFre6yL2zbkI0kocXqfoQx3rB(
gOsWYOYEslvJujTNkdvgdOIaKeaL79jzgYohrAfSyMVyL5pd9nwHQTrRZtVlOWZFJipsTsTTYDIDPEB8
mzvxXk3jB)j56u8Y2Qh5b4NskmJVuI0J8nxJMp57uY1w3qDlTxNSrExTd9YrIGoAlsUsDD1oklhed4WM
Luu8sgtm259Pvs6G2ZwdpFjNDApsent6iqvgkFVczkrS8apv6hfmNyvFsDKJnrL(wMLRXWVStdhsHqly
FQv2y)zxLvq6Yl9cft3zNWSmtepWw9TuMlDvevlSUvo8BuZ(eNTMGUlLq3IzQirpPXG8TjKagFhzPh5R
cQjV5QCMIYvACzPDFLjXx(580rj8ktLTtOugNXtRsQzn8ouDglvNGYtKSia3x6GgFhCPbuV3NqvqSjjW
Y2KnATbu7i1QaCIRlkedZc9K8OowHXQxq6myzFB1RcBES80siicB2ik90(sPQkSoVCfHxwI7jSLOyjsr
ivJoJyjZxhuHbzQdT6OruR78T1mcDsdpHgQSdwErzGqFP7DuVXeFVjXecfd7Qj)0vRcQPMhPUxs1ngZQ
9FNVdPzOIQ9vN2I)nQ(WQXKuvItUS6u3ZRkBYHLWw74M1ictxqouQ5kmkZcSFPQiVSzG5tex1cqA4TTv
ZymjNZG6nPo3rEnhbXydSeDlknYjTiduH9fuBTKSNqMUGvB9OZptbiOVCww6gHMss9B6vRgcpr5R1lhE
fQEwMLh5kMiwJK3xJCGDZnJu8UzLJ)LroXM5IlLy56(DXxqEcsud49kq9Aef14TvsaK0NQop194JsmPM
mK2Oh4uJDF2XBt)HLl2DCTCrJl324xSAZQdp63UA9r6Iq8VUz9NhE)6XF5lV7I3Cowg6M4CKU(a)RndF
8hrcuF)(JVFZ4UBhVgJzZHFmxuH90HH6T9FpdPV))9MR(rp8Gg4XN)BXqD8YTF(hrkn8iiXRVoIRwpSF
)p(OF(vp)rx8l0fGIUYdGd(1J4Tp4IlE2ROH8w6cwq3BQ4xpMZE0lF15p9nNFzguNnUzA7D47e86(gW7
fV6HpVaV1tx9H8T1(0X8Yh8Kh9l0TZ4289x93VCp4YN(8ZF2tE6L5LB4WDFyB5w8s3H3th8fV6jVHa2f
t3EmFVJ(Dl2RoNWMxoT9dKAZx)5fp9bV8bN)l5Bu3M8L2291d51p4fp4SNrJHUuvxtxfiZVJtF2fV5zN
rOlDe9xs3W9VAak)7PmRZ8xWNFbyE)3NWSn)oU9RV4zpAbXLE9UvJ7pW)1nz30XpYNe)kV7YLYrSTxAK
CVCiV7L8o6Ld0BVuzC)sP7K9sti6xkNm)EjNz6vLtLt)s5GL1t4IWXiZMsFQ6LD8RxoEV9l5SlZRfLNy
(n(c0khyTEXByzWTz4u2i6Ej(FVuUBVCm2i0QeXVGGUcQwoGleKkHs7xkDKlpHCQO5NYNb7mfL3XF6DL
Ml2x(BjuwJHiTYz6Vxc40lNMYEXpsEjYT37eUhhwH(A5m43l7w3jC0ssp5rL3b1(zIIs3mZiuwh)YLsB
zz4N8cXiVl3CM(LCoG8W(AeIt0PxYyNqJ85lN(w5yvCctLluNgFSSoL03MZc4EOmNpYNA)8RYvNllTiZ
DPcXvmWVCP0NPtrJs9SeoMpOY9lLcb4p6leCo3(cjvuQQuPQUx6oJauEE2c7PuGEMgBob55tjmnRYoB3
Vuo1ofrd5R7YtO78wm3pxERZuvEubp3qdMgkCPs5n9s7W6NRevXNYor1lhbn6BL2U3x1DKJbq)CT0YgB
3p3oOQNZhC2E9VnuNQ3vcPtRE(IeCcXntWu2VR(zYJYjQRFPKDFEu5JZw)xlqYWxfaLoL2pZcmFnCOxu
AbygvLyCxUuocbYsvyPX8XjSFU2SklkNX2(LYgY3p3Lu5ojLbwoNDbySiWyeDSaRHlgc85(Vxk3Kq4sX
00WkvH1lhs9c7EwiHlNRDZ1J0ptMxDDRU15nnHgFPSS(L8zJPxoP(9ZuPMrzLtRe9uzZwko4lbEUCjFd
cOpNp(t9l5tZpHg5ZwyVKax)s5aq0Zcv2EH6GDgDyT4C)U6NRvezp(85OlJcpTYlkNdUEPDh9YXdPFUc
yP)M9lLcf6lYk2OT0(bIYlND3EPMP(zr2Kge1xIgPRKHjYSPArFlNd0LZ52LZmF)CdXQ0ryTZKtvgF5G
R2Zfw0ptEPorkNZS8IZ58v0rOZP1mCtk4QFw0y(0(2ZgMFv0WQIqfTv9M5(mB70ykPk0Af8x9ffKhkUw
0SdQk0QlzU3(cSSNYPuRWktqm5l4JgzJ1g1GO1atLJMr(jXfhVf69lLkc7N5mq9EppcB(kk2px0voG69
Zu8vf3cQX(CZ1m1ppne(mHYWWlpef4NuZGw1YQOJwJevJ2w2038tIVCEJh7N5vsJcKtJCA91pE9WTp8U
HT3oo7poEiVYfV9zVC5I)RY4U5(mOF)FaNUA6JF(Ybus0H95CAx((LL8xx()9d

**Comment by mbattersby** [comment]
If you want the "smart" behavior you need to use the "Smart Priority Mount" action in your rules.

The "Random Mount" action just randomly chooses from the mounts, with no concern for the current situation.

That's why you are getting the "wrong" mounts.

**Comment by mbattersby** [comment]
Also, really appreciate you uploading the bug report. Makes things very easy to diagnose. Thanks!

**Comment by mbattersby** [comment]
Alternatively (since I see you have a manual "exclude ground mounts in flying situations" rule), you could add more similar rules to limit mounts for ground, aquatic and dragonflying situations. I think you are better off removing that rule and using the "Smart Priority Mount" action though.

**Comment by RetroFrito** [comment]
The reason I did a bug report is because out of all my other "flying" mounts, those two  are the ones that keep popping up when I'm in ground-only situations. I would think that if it was something else, other flying mounts would also be popping up.

**Comment by mbattersby** [comment]
Those are the only 2 in your group. That's why they pop up and the others don't, because it's randomly choosing between the mounts in the group your rule applies to.

**Comment by RetroFrito** [comment]
OK I don't think I am familiar with your earlier suggestion "Smart Priority Mount" I am looking through rules and don't see that.

**Comment by RetroFrito** [comment]
So maybe I just need a bit of coaching on how to use this addon... but when I am in no-fly zones, or if I do not have flight yet (for example I do not have flight unlocked in Shadowlands), I would like to exclude flying mounts. I tried a rule "Not flyable area - Smart Priority Mount - Ground" but then it ignored the mounts I'd selected for that group/class. What do I need to do to limit to ground mounts when I can't fly, and also limit to mounts I've selected for that group?

**Comment by mbattersby** [comment]
https://github.com/xod-wow/LiteMount/wiki/Rules

**Comment by RetroFrito** [comment]
I think I figured it out. I had to alter the rule I'd described before. Instead of "Not flyable area - Smart Priority Mount - Ground" I changed it to "Not flyable area - Exclude Mounts - Type: Flying" moved it near the top and that seems to have fixed it. Thanks so much for all your help on this.
